docs/DesignDocs/CustomAttestation.md

```diff
@@ -646,9 +646,29 @@ typedef void* oe_sgx_plugin_opt_params;
  * Helper function that returns the SGX attester that can then be sent to
  * `oe_register_attester`.
  *
+ * @deprecated Instead, use oe_sgx_plugin_local_attester()
+ * for local attestation attester or
+ * oe_sgx_plugin_ecdsa_p256_attester() for remote attestation attester.
+ *
  * @retval A pointer to the SGX attester. This function never fails.
  */
 oe_attester_t* oe_sgx_plugin_attester(void);
+
+/**
+ * Helper function that returns the SGX local attester that can then be sent to
+ * `oe_register_attester`.
+ *
+ * @retval A pointer to the SGX attester. This function never fails.
+ */
+oe_attester_t* oe_sgx_plugin_local_attester(void);
+
+/**
+ * Helper function that returns the SGX ECDSA P256 attester that can then be sent to
```

Comment:
Why call this the ECDSA P256 attester? Keep in mind that most developers will not be familiar with this as a reference to the DCAP-based remote attestation workflow. It's also not clear why the other plugin is called 'local_attester' if this one is referred to by the signature algorithm. 

In both cases, the doxygen comments need to be much more descriptive about what the attesters/verifiers actually do, which will help make up for only so much information you can put in a function name.

Reply:
I have the similar concerns on the local/remote splitting and the naming here. I may need to think more to appraise the pros and cons of this local/remote splitting. But my feeling is that even if we choose to go with the local/remote splitting approach, we still need better names for those functions.